.github/workflows/CI.yml

```diff
@@ -91,6 +91,9 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
       - uses: actions/checkout@v3
+      - name: Update version
+        run: |
+          echo -e "package api\n\nconst version_str=\"${GITHUB_REF_NAME}\"" > int/api/server_version.go
```

Comment:
should we do this also in CD workflow?

Reply:
You are right. I missed that. I thought we were using the CI workflow in the CD one to build the binaries. But we don't.